Chavah.NetCore/Common/Extensions/QuartzAppBuilderExtensions.cs

```diff
@@ -23,12 +21,12 @@ public static IApplicationBuilder UseQuartzForEmailRetry(this IApplicationBuilde
             // Use our QuartzJobFactoryWithIoC so that EmailRetryJob can be instantiated with dependencies.
             var scheduler = Quartz.Impl.StdSchedulerFactory.GetDefaultScheduler().Result;
             scheduler.JobFactory = new QuartzJobFactoryWithIoC(app.ApplicationServices);
-            var emailSettings = app.ApplicationServices.GetRequiredService<IOptions<AppSettings>>();
+            var emailOptions = app.ApplicationServices.GetRequiredService<IOptionsMonitor<EmailOptions>>();
```

Comment:
Why IOptionsMonitor?

Reply:
If the values change in the providers the application doesn't have to be restarted to pick up the new value.